src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -230,10 +237,11 @@ void create_actor(SchedulingAlgorithmState *algorithm_state,
                   LocalSchedulerClient *worker) {
   /* This will be freed when the actor is removed in remove_actor. */
   LocalActorInfo *entry = (LocalActorInfo *) malloc(sizeof(LocalActorInfo));
+  memset(entry, 0, sizeof(LocalActorInfo));
```

Comment:
Why did you introduce the calls to `memset`? I imagine it crashes without the `memset`?